core/templates/dev/head/pages/community-dashboard-page/translation-opportunities/translation-opportunities.directive.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the translation opportunities.
+ */
+
+require(
+  'pages/community-dashboard-page/opportunities-list/' +
+  'opportunities-list.directive.ts');
+
+require(
+  'pages/community-dashboard-page/services/' +
+  'contribution-opportunities.service.ts');
+require(
+  'pages/exploration-editor-page/translation-tab/services/' +
+  'translation-language.service.ts');
+
+var oppia = require('AppInit.ts').module;
+
+oppia.directive(
+  'translationOpportunities', ['UrlInterpolationService', function(
+      UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/community-dashboard-page/translation-opportunities/' +
+      'translation-opportunities.directive.html'),
+      controllerAs: '$ctrl',
+      controller: [
+        '$scope', 'ContributionOpportunitiesService',
+        'TranslationLanguageService', function(
+            $scope, ContributionOpportunitiesService,
+            TranslationLanguageService) {
+          var ctrl = this;
+          ctrl.opportunities = [];
+          ctrl.isLoading = true;
```

Comment:
Change to XXXIsLoading (where XXX is something appropriate -- "component"? "page"?) so that it doesn't get misinterpreted as a function.